spring-xd-shell/src/test/java/org/springframework/xd/shell/command/ModuleListTests.java

```diff
@@ -16,18 +16,16 @@
 
 package org.springframework.xd.shell.command;
 
-import static com.google.common.collect.Lists.newArrayList;
-import static com.google.common.collect.Maps.transformValues;
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasEntry;
-import static org.hamcrest.Matchers.hasSize;
-import static org.junit.Assert.assertThat;
+import static com.google.common.collect.Lists.*;
```

Comment:
Yeah the default in IDEA is to squash 10 or so imports into a wild card. This can be changed in the settings.